test/MUXControls.Test/MSTest/BuildAppX.cmd

```diff
@@ -1,14 +1,14 @@
 @echo off
 
 if "%DevEnvDir%" == "" (
-    if exist "%ProgramFiles(x86)%\Microsoft Visual Studio\2017\Enterprise" (
-        call "%ProgramFiles(x86)%\Microsoft Visual Studio\2017\Enterprise\Common7\Tools\VsDevCmd.bat"
+    if exist "%ProgramFiles(x86)%\Microsoft Visual Studio\2019\Enterprise" (
```

Comment:
We should use "vswhere" to find VisualStudio -- you can see examples in DevCmd and others.

Reply:
Good call, will do.